apisix/plugins/response-rewrite.lua

```diff
@@ -124,7 +127,7 @@ function _M.body_filter(conf, ctx)
         return
     end
 
-    if conf.body then
+    if conf.body and #conf.body > 0 then
```

Comment:
Do we need the check here? We already check it in `check_schema`.

Reply:
@spacewander yes, this additional check is indeed redundant. Done. Thank you.